cpu/cortexm_common/thread_arch.c

```diff
@@ -234,6 +238,15 @@ char *thread_arch_stack_init(thread_task_func_t task_func,
     return (char*) stk;
 }
 
+/* This function returns the number of bytes used on the ISR stack */
+int thread_arch_isr_stack_usage(void)
+{
+    uint32_t  *ptr = &_sstack;
+    while (*(ptr++) == STACK_CANARY_WORD) {
+    }
+    return ISR_STACKSIZE - (ptr - &_sstack);
```

Comment:
I ~~have no~~ did not have a clue on how the ISR stack works on AVR, but I found this answer on stackexchange:
http://electronics.stackexchange.com/questions/149578/how-does-the-call-stack-operated-during-an-interrupt-on-avr

So it seems like the AVR does not have a separate ISR stack. I guess it should return 0 for both size and usage in that case (or -1 if you prefer that)